COBOL Programming Course #1 - Getting Started/COBOL Programming Course #1 - Getting Started.md

```diff
@@ -3192,58 +3212,24 @@ A PERFORM with UNTIL phrase is a conditional expression.  In the UNTIL phrase fo
 
 
 ```
-WORKING-STORAGE.
-01 FACIAL-EXP    PIC X(11) VALUE SPACES.
-   88 HAPPY      VALUE 'HAPPY'.
-....
-....
-PROCEDURE DIVISION.
-....
-....
-PERFORM SAY-SOMETHING-DIFFERENT UNTIL HAPPY
+PERFORM SAY-SOMETHING-DIFFERENT BY FACIAL-EXP UNTIL 'HAPPY'
 END-PERFORM.
 ```
 
-*Example 5. PERFORM statement with 88-level conditional name*
+*Example 5. PERFORM statement*
 
 
-It is also possible to use PERFORM statement without the use of a 88-level conditional name, observe Example 6.
-
-```
-WORKING-STORAGE.
-01 FACIAL-EXP    PIC X(11) VALUE SPACES.
-....
-....
-PROCEDURE DIVISION.
-....
-....
-PERFORM SAY-SOMETHING-DIFFERENT UNTIL FACIAL-EXP = "HAPPY"
-END-PERFORM.
-```
-
-*Example 6. PERFORM statement without 88-level conditional name*
- 
-
 ### SEARCH statements
 
-The SEARCH statement searches a table for an element that satisfies the specified condition and adjusts the associated index to indicate that element.  Tables, effectively an array of values, are created with an OCCURS clause applied to WORK-STORAGE data-names.  A WHEN clause is utilized in SEARCH statements to verify if the element searched for satisfies the specified condition.  Assuming FACIAL-EXP has many possible values, then SEARCH WHEN is an alternative conditional expression, observe Example 7.
+The SEARCH statement searches a table for an element that satisfies the specified condition and adjusts the associated index to indicate that element.  Tables, effectively an array of values, are created with an OCCURS clause applied to WORK-STORAGE data-names.  A WHEN clause is utilized in SEARCH statements to verify if the element searched for satisfies the specified condition.  Assuming FACIAL-EXP has many possible values, then SEARCH WHEN is an alternative conditional expression, observe Example 6.
 
 
 ```
-WORKING-STORAGE.
-01  FACIAL-EXP-TABLE REDEFINES FACIAL-EXP-LIST.
-    05  FACIAL-EXP  PIC X(11) OCCURS n TIMES INDEXED BY INX-A.
-        88  HAPPY VALUE "HAPPY".
-....
-....
-PROCEDURE DIVISION.
-....
-....
 SEARCH FACIAL-EXP
-WHEN HAPPY(INX-A) DISPLAY 'I am glad you are happy'
+WHEN 'HAPPY' STOP RUN
 END-SEARCH
 ```
-*Example 7. SEARCH WHEN statement*
+*Example 6. SEARCH WHEN statement*
```

Comment:
Also, if you can reverse this changes, that will be great :) It was added in #228 to fix an issue.

Reply:
sorry, this was a stupid mistake on my part